test/output/aaplBollinger.svg

```diff
@@ -13,80 +13,70 @@
       white-space: pre;
     }
   </style>
-  <g aria-label="y-axis" aria-description="↑ Close" transform="translate(40,0)" fill="none" text-anchor="end" font-variant="tabular-nums">
```

Comment:
The aria-description here, with the axis label, is now lost, is this expected?

Reply:
It’s redundant with the label, right? I’m not sure how much value it adds but we could try to find a new place for it. There’s no longer a single G element for the axis since it is now multiple marks.

Reply:
Maybe we don't need it, but I wanted to mention it if it wasn't intentional. BTW maybe this is an indication that nested/composite marks should return nested layers?